span-normalizer/src/test/java/org/hypertrace/core/spannormalizer/JaegerSpanNormalizerTest.java

```diff
@@ -4,19 +4,31 @@
 import com.typesafe.config.ConfigFactory;
 import io.jaegertracing.api_v2.JaegerSpanInternalModel.Process;
 import io.jaegertracing.api_v2.JaegerSpanInternalModel.Span;
+import io.micrometer.core.instrument.Timer;
 import java.io.File;
 import java.lang.reflect.Field;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.Random;
 import org.hypertrace.core.datamodel.RawSpan;
+import org.hypertrace.core.serviceframework.metrics.PlatformMetricsRegistry;
 import org.hypertrace.core.span.constants.RawSpanConstants;
 import org.hypertrace.core.span.constants.v1.JaegerAttribute;
 import org.hypertrace.core.spannormalizer.jaeger.JaegerSpanNormalizer;
 import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
 public class JaegerSpanNormalizerTest {
+  private final Random random = new Random();
+
+  @BeforeAll
+  static void initializeMetricRegistry() {
```

Comment:
Do you need to clear the status of this and isn't it better to do on BeforeEach and clear in AfterEach?

Reply:
The test is using the registry similar to a regular service would use it. It's initialized once and each test is like a separate thread/method. Hence, no need to clear the registry for this test.

Reply:
ok.